draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -279,6 +281,12 @@ Implementers should be aware that using the same client attestation across multi
 
 The guidance provided by {{RFC7519}} and {{RFC8725}} applies.
 
+Replay attack prevention MUST be considered by the protocols using this client authentication method. Possible mechanisms comprise:
+
+- The client uses "jti" (JWT ID) claims for the Client Attestation PoP JWT and the authorization server maintains a list of used (seen) "jti" values for the time of which the JWT would be considered valid based on the applicable "exp" claim. If any Client Attestation PoP JWT would be replayed, the authorization server would recognize the "jti" and respond with an authentication error.
```

Comment:
agreed, then we shoudl mandate the presence of `jti` in the WIA as it was before this PR